website/source/docs/providers/cloudstack/r/affinity_group.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: "cloudstack"
+page_title: "CloudStack: cloudstack_affinity_group"
+sidebar_current: "docs-cloudstack-resource-affinity-group"
+description: |-
+  Creates an affinity group.
+---
+
+# cloudstack\_affinity\_group
+
+Creates an affinity group.
+
+## Example Usage
+
+```
+resource "cloudstack_affinity_group" "default" {
+  name = "myGroup"
+  type = "host anti-affinity"
+  project = "myProject"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the affinity group. Changing this 
+    forces a new resource to be created.
+
+* `type` - (Required) The affinity group type. Changing this 
+    forces a new resource to be created.
+
+* `description` - (Optional) The description of the affinity group.
+
+* `project` - (Optional) The name or ID of the project to register this
+    affinity group to. Changing this forces a new resource to be created.
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `id` - The id of the affinity group.
+* `name` - The name of the affinity group.
```

Comment:
In order for this new file to be shown in the index, it should also be added here: https://github.com/hashicorp/terraform/blob/master/website/source/layouts/cloudstack.erb